distributed/protocol/compression.py

```diff
@@ -124,16 +124,22 @@ def zstd_decompress(data):
     }
 
 
-default = dask.config.get("distributed.comm.compression")
-if default != "auto":
-    if default in compressions:
-        default_compression = default
+def get_default_compression():
+    default = dask.config.get("distributed.comm.compression")
+    if default != "auto":
+        if default in compressions:
+            return default
+        else:
+            raise ValueError(
+                "Default compression '%s' not found.\n"
+                "Choices include auto, %s"
+                % (default, ", ".join(sorted(map(str, compressions))))
+            )
     else:
-        raise ValueError(
-            "Default compression '%s' not found.\n"
-            "Choices include auto, %s"
-            % (default, ", ".join(sorted(map(str, compressions))))
-        )
+        return default_compression
+
+
+get_default_compression()
```

Comment:
For my own knowledge, we're just calling `get_default_compression` here to potentially trigger the `ValueError` above?

Reply:
Yes.  I'm mostly mimicing the previous behavior in order to not have to think too much about it :)

Reply:
+1